### PR TITLE
Create maven project

### PR DESCRIPTION
**Pull request by AlasDiablo**
Signed-off-by: AlasDiablo <25723276+AlasDiablo@users.noreply.github.com>

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -0,0 +1,16 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+    <modelVersion>4.0.0</modelVersion>
+
+    <groupId>org.example</groupId>
+    <artifactId>Projet ACL</artifactId>
+    <version>1.0-SNAPSHOT</version>
```
Comment (clementsven) [inline pom.xml:9]: bn

**Review by clementsven — APPROVED** [review]
b

File: Projet ACL.iml [hunk 2]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
+    <exclude-output />
+    <content url="file://$MODULE_DIR$" />
+    <orderEntry type="inheritedJdk" />
+    <orderEntry type="sourceFolder" forTests="false" />
+  </component>
+</module>
```
Comment (Emeline45) [inline Projet ACL.iml:9]: test

File: Projet ACL.iml [hunk 3]
```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<module type="JAVA_MODULE" version="4">
+  <component name="NewModuleRootManager" inherit-compiler-output="true">
```
Comment (Emeline45) [inline Projet ACL.iml:3]: test2

Comment (NikolasBdn) [inline Projet ACL.iml:3]: cool

**Pull request closed without merging**